ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/ClustersCoverageTable.tsx

```diff
@@ -21,21 +21,22 @@ import { SearchIcon } from '@patternfly/react-icons';
 import useRestQuery from 'hooks/useRestQuery';
 import useURLPagination from 'hooks/useURLPagination';
 import useURLSearch from 'hooks/useURLSearch';
-import { getComplianceClusterScanStats } from 'services/ComplianceEnhancedService';
+import { getAllClustersCombinedStats } from 'services/ComplianceEnhancedService';
```

Comment:
An observation, not a blocker to merge, maybe Visual Studio Code eagerly added this `import` statement and `getCompliancePfClassName` below out of alphabetical order?

Reply:
What is the alphabetical order by file for this section? Do you prefer we alphabetize the based off the directory ignoring casing?

So:
```